src/test/scala/com/dwolla/lambda/cloudflare/requests/ResourceRequestFactorySpec.scala

```diff
@@ -27,6 +28,8 @@ class ResourceRequestFactorySpec(implicit ee: ExecutionEnv) extends Specificatio
     val mockClient = Client.fromHttpApp(HttpRoutes.empty[IO].orNotFound)
 
     val customResourceType = "Custom::Tester".asInstanceOf[ResourceType]
+
+    implicit val contextShift = IO.contextShift(ee.executionContext)
```

Comment:
An implicit ContextShift was required here. Otherwise I encountered compilation errors around a ContextShift[cats.effect.IO] not existing.